doc/ref_model/chapters/chapter08.rst

```diff
@@ -879,3 +879,26 @@ Comparison of Deployment Topologies and Edge terms
 +--------+--------+--------+--------+--------+--------+--------+--------+-------+-------+-------+-------+-------+------+
 
 **Table 8-6:** Comparison of Deployment Topologies
+
+O-RAN alignment and interaction
+-------------------------------
+
+O-RAN is an operator alliance group with members from the major Telco Operators, Vendors and other interested eco system participants around the specification of Open Radio Access Networks.  Its task is to cloudify the 3GPP specified RAN Network Functionalities with multi-vendor open interfaces in between the Network Functions, the Cloud Infrastructure and the management them. The Service Management and Orchestration (SMO) of multiple O-Clouds is also during specification including a framework for 3:rd party applications (rApps). There are also a few other open interfaces that are aimed to be specified e.g. for Radio Layer 1 HW Accelerators and some low level Radio functions. 
+
+On a high level O-RAN covers similar specification grounds as what Anuket do, but there are some noteworthy differences both on specification level and on the aim for how O-Clouds are realized. O-RAN specifies how management and orchestration of the Network Functions and Cloud Infrastructure shall be done with a set of internal Services that also have a set of interface specifications for how the rApps could enhance the management functionality. O-RAN have also articulated that O-Clouds can be distributed over multiple Cloud Sites that are stitched over an externally specified WAN interconnect transport that is not part of the O-RAN.
```

Comment:
```suggestion
On a high level, O-RAN covers similar specification grounds as what Anuket do, but there are some noteworthy differences both on specification level and on the aim for how O-Clouds are realized. O-RAN specifies how management and orchestration of the Network Functions and Cloud Infrastructure shall be done with a set of internal Services that also have a set of interface specifications for how the rApps could enhance the management functionality. O-RAN have also articulated that O-Clouds can be distributed over multiple Cloud Sites connected over an externally-specified WAN interconnect transport that is not part of the O-RAN.
```

Reply:
Agree on the M call not to change the original text.